common/src/main/java/org/ergoplatform/appkit/InputBoxesSelectionException.java

```diff
@@ -0,0 +1,45 @@
+package org.ergoplatform.appkit;
+
+import java.util.Map;
+
+public class InputBoxesSelectionException extends RuntimeException {
```

Comment:
What is the motivation to introduce this wrappers around ergo-wallet exceptions?

Reply:
It enables clients to access the original error information and beautify it for the user. Most important is to show ERG amounts instead of nanoergs for NotEnoughErgsException, and additionally showing a list instead of a serialized map for tokens.